draft-ietf-dnsop-svcb-https.md

```diff
@@ -1711,7 +1711,7 @@ Global Scoped Entry Registry:
 
 # Acknowledgments and Related Proposals
 
-There have been a wide range of proposed solutions over the years to
+There has been a wide range of proposed solutions over the years to
```

Comment:
This seems like at least a debatable point of grammar.

Reply:
Agreed.  I think either would be fine, and I'm inclined to let the RFC Editor sort it out at this point.